frontend/src/routes/(authenticated)/project/create/+page.svelte

```diff
@@ -58,7 +58,13 @@
     let typeCode = typeCodeMap[$form.type] ?? 'misc';
     let policyCode = policyCodeMap[$form.retentionPolicy] ?? '';
     if (policyCode) policyCode = `-${policyCode}`;
-    $form.code = `${$form.languageCode}${policyCode}-${typeCode}`;
+    form.update(
+      ($$form) => {
+        $$form.code = `${$$form.languageCode}${policyCode}-${typeCode}`;
+        return $$form;
```

Comment:
why the $$? are we accessing a svelte store?

Reply:
Good point. I was just trying to make it distinct, but I guess it doesn't really matter if it has the same name, in fact using the same name prevents us from accidentally referencing the wrong form object.